neurodocker/templates/gentoo-portage/repos.conf/gentoo

```diff
@@ -0,0 +1,6 @@
+[gentoo]
```

Comment:
all the files should be produced by the Dockerfile script, not required to be copied since will need then a copy etc... mess.  See https://github.com/nimh-dsst/dsst-defacing-pipeline/pull/41/files